jitbuilder/release/src/TransactionalOperations.cpp

```diff
@@ -0,0 +1,131 @@
+/*******************************************************************************
```

Comment:
Don't remove it, but we should devise a test for it. See my other comment about opening an issue to extend the test case.